clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/HealthState.groovy

```diff
@@ -2,5 +2,5 @@
 package com.netflix.spinnaker.clouddriver.model
 
 enum HealthState {
-  Up, Down, Unknown, Starting, OutOfService
+  Up, Down, Unknown, Starting, OutOfService, Succeeded, Failed
```

Comment:
I'd think there are similar states/results defined for pipelines.

On Sat, Mar 12, 2016 at 9:26 AM Lars Wander <notifications@github.com>
wrote:

> In
> clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/HealthState.groovy
> <https://github.com/spinnaker/clouddriver/pull/409#discussion_r55922088>:
>
> > @@ -2,5 +2,5 @@
> >  package com.netflix.spinnaker.clouddriver.model
> >
> >  enum HealthState {
> > -  Up, Down, Unknown, Starting, OutOfService
> > +  Up, Down, Unknown, Starting, OutOfService, Succeeded, Failed
>
> I am up for moving this into a separate enum. Maybe JobState? Can be Running,
> Succeeded, Failed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spinnaker/clouddriver/pull/409/files#r55922088>.
>